h3/src/connection.rs

```diff
@@ -110,8 +110,10 @@ where
             .insert(SettingId::MAX_HEADER_LIST_SIZE, max_field_section_size)
             .map_err(|e| Code::H3_INTERNAL_ERROR.with_cause(e))?;
 
-        // Grease Settings (https://httpwg.org/specs/rfc9114.html#rfc.section.7.2.4.1)
         if grease {
+            //= ci/compliance/specs/rfc9114.txt#7.2.4.1
```

Comment:
I recommend linking to the page on rfc editor so it's easier to click the link and read the section

```suggestion
            //= https://www.rfc-editor.org/rfc/rfc9114#section-7.2.4.1
```

Reply:
My bad. Will fix this real quick.

Reply:
> My bad. Will fix this real quick.

Ok thanks.